server/BusinessLogic/ProductManagement/DTOs/CreateProductDTO.cs

```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace CoffeeShop.BusinessLogic.ProductManagement.DTOs;
+
+public sealed record CreateProductDTO(
+    string Name,
+    string Description,
+    [RegularExpression(@"^\d+(\.\d{1,2})?$", ErrorMessage = "Price can have up to two decimal places.")]
+    decimal Price,  
+    string ImageUrl,
```

Comment:
I suggest to add image url later since we will need to upload the image which we do not have now